tests/rhoai_mcp/constants.py

```
@@ -1,3 +1,7 @@
+# rhoai-mcp is not managed by an Operator; use the floating tag to test the latest build from the MCP catalog.
+RHOAI_MCP_ODH_STABLE: str = "quay.io/opendatahub/odh-rhoai-mcp:odh-stable"  # noqa: IMG001
+RHOAI_MCP_RHOAI_VERSION: str = "registry.redhat.io/rhoai/odh-rhoai-mcp-rhel9:rhoai-3.6-ea.1"  # noqa: IMG001
```

Ruff (0.16.1)

[warning] Invalid rule code in `# noqa`: IMG001

Add non-Ruff rule codes to the `lint.external` configuration option

(RUF102)